frontend/src/views/Holdings.vue

```diff
@@ -0,0 +1,257 @@
+<template>
+  <HeadlineActions
+    :headline="$t('holdings.headline')"
```

Comment:
Is this file just a test?

Reply:
Oops, added a bit early, I'm still working on it.  

I might have something later today for the holdings.

Reply:
😂😂 awesome! Im looking forward to it 👍